tools/ios.toolchain.cmake

```diff
@@ -151,10 +151,10 @@
 cmake_minimum_required(VERSION 3.8.0)
 
 # CMake invokes the toolchain file twice during the first build, but only once during subsequent rebuilds.
-#if(DEFINED ENV{_IOS_TOOLCHAIN_HAS_RUN})
-#  return()
-#endif()
-#set(ENV{_IOS_TOOLCHAIN_HAS_RUN} true)
+if(DEFINED ENV{_IOS_TOOLCHAIN_HAS_RUN})
+  return()
+endif()
+set(ENV{_IOS_TOOLCHAIN_HAS_RUN} true)
```

Comment:
Pretty sure you don't want this. The maintainer of the toolchain file doesn't remember why he added this and refuses to revert it, and it has always been a problem for us. See https://github.com/leetal/ios-cmake/pull/198 and https://github.com/leetal/ios-cmake/issues/146.

Reply:
Oh, right, thanks!

Reply:
So it didn't actually update at all, ok I'm stupid.